heapster.go

```diff
@@ -35,8 +35,7 @@ import (
 )
 
 var (
-	argPollDuration    = flag.Duration("poll_duration", 10*time.Second, "The frequency at which heapster will poll for stats")
-	argStatsResolution = flag.Duration("stats_resolution", 5*time.Second, "The resolution at which heapster will retain stats. Acceptable values are in the range [1 second, 'poll_duration')")
+	argStatsResolution = flag.Duration("stats_resolution", 5*time.Second, "The resolution at which heapster will retain stats.")
```

Comment:
I don't mean removing sampling logic here at all but rather simplifying since we need to take exactly one samples per time interval as for now. It might make sense in the future to support such call in cadvisor (give me the first sample from specified time range) which will significantly reduce the amount of data being sent between heapster and cadvisor. WDYT?